<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StuSignInDlg</class>
 <widget class="QWidget" name="StuSignInDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>312</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>440</width>
    <height>290</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>440</width>
    <height>312</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>答题器</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#StuSignInDlg{
	background-color: rgb(211, 211, 211);
}	</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="label_5">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>10</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>10</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/widget/img/widget/back.png) 0 0 194 0;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_4">
     <property name="minimumSize">
      <size>
       <width>14</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>14</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/widget/img/widget/back.png) 6 352 18 0;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="CWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">#widget
{
	background-color: rgb(255, 255, 255);
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(82, 204, 144);

</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="leftMargin">
          <number>27</number>
         </property>
         <property name="rightMargin">
          <number>10</number>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);
font: 16px &quot;微软雅黑&quot;;</string>
           </property>
           <property name="text">
            <string>签到</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnClose">
           <property name="minimumSize">
            <size>
             <width>18</width>
             <height>18</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>18</width>
             <height>18</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{	
	border-image: url(:/Login/img/Login/close_green_Normal.png);
}

QPushButton:hover{		
	border-image: url(:/Login/img/Login/close_green_Click.png);
}

QPushButton:pressed{
	border-image: url(:/Login/img/Login/close_green_Normal.png);
}

QPussButton:disable{
	border-image: url(:/Login/img/Login/close_green_Normal.png);
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>48</number>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>30</number>
        </property>
        <property name="horizontalSpacing">
         <number>0</number>
        </property>
        <property name="verticalSpacing">
         <number>10</number>
        </property>
        <item row="3" column="0">
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0" colspan="3">
         <widget class="QLabel" name="labCountDown">
          <property name="styleSheet">
           <string notr="true">color:rgba(68,68,68,255);
font: 14px &quot;微软雅黑&quot;;</string>
          </property>
          <property name="text">
           <string>您有&lt;font color=#FF0000&gt;30&lt;/font&gt;秒的时间进行签到</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="btnStartSignIn">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>30</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
border-top-left-radius : 15px; 
border-top-right-radius : 15px; 
border-bottom-left-radius : 15px; 
border-bottom-right-radius : 15px; 
background-color: rgb(82, 204, 144);
font: 12px &quot;微软雅黑&quot;;
color: rgb(255, 255, 255);
}

QPushButton:hover{
border-top-left-radius : 15px; 
border-top-right-radius : 15px; 
border-bottom-left-radius : 15px; 
border-bottom-right-radius : 15px; 
background-color: rgb(66, 198, 97);
font: 12px &quot;微软雅黑&quot;;
color: rgb(255, 255, 255);
}</string>
          </property>
          <property name="text">
           <string>签到</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="3">
         <widget class="QLabel" name="labStatistics_2">
          <property name="styleSheet">
           <string notr="true">color:rgba(68,68,68,255);
font: 14px &quot;微软雅黑&quot;;</string>
          </property>
          <property name="text">
           <string>主持人发起了签到</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>32</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="label_6">
     <property name="minimumSize">
      <size>
       <width>14</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>14</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/widget/img/widget/back.png) 10 0 18 352;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QLabel" name="label_7">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>22</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>22</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/widget/img/widget/back.png) 182 0 0 0;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CWidget</class>
   <extends>QWidget</extends>
   <header>cwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../VLive/VLive.qrc"/>
 </resources>
 <connections/>
</ui>
